installation_docs/manual/linux.md

```diff
@@ -44,7 +44,7 @@ cd CircuitVerse
      ```
 - [Node.js 16.x](https://nodejs.org/it/download)
      ```bash
-     curl -sL https://deb.nodesource.com/setup_16.x | sudo bash
+     curl -sL https://deb.nodesource.com/node_16.x | sudo bash
```

Comment:
Have you tested this ?

Reply:
Yes sir , i have tested this code on my local setup in docker and also tested in gitpod, but container build fail in code Space.

Reply:
Also as I am new here I don't know how to test for production docker build . can You plz help

Reply:
I guess for the install command in linux.md this would not work you need to use something like 
```
sudo apt-get update && sudo apt-get install -y ca-certificates curl gnupg
curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | sudo gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg
NODE_MAJOR=16
echo "deb [signed-by=/etc/apt/keyrings/nodesource.gpg] https://deb.nodesource.com/node_$NODE_MAJOR.x nodistro main" | sudo tee /etc/apt/sources.list.d/nodesource.list
sudo apt-get update && sudo apt-get install nodejs -y
```

as provided in https://deb.nodesource.com/

Reply:
Also please join the Slack channel so we can talk regarding the testing of docker parts